solution/backend/regulations/templates/regulations/partials/header/reg-title.html

```diff
@@ -1,9 +1,12 @@
 {% load static %}
 
-{% if part %}
+
     <div class="reg-title">
```

Comment:
Having `reg-title` on every page is a fine solution to this, but it adds a gray left border to the header:

<img width="839" alt="Screen Shot 2022-09-15 at 09 12 47" src="https://user-images.githubusercontent.com/1651907/190413055-3f76d20c-276b-435f-b649-8f5351e0ad3c.png">

That border is generally meant to separate the site title ("Medicaid & CHIP eRegulations") and the reg title on the Part TOC/Reader view pages (ex: "42 CFR Part 433").  It's there's no reg title, the border should be hidden.

My vote would be to hide it on the homepage/about page/search results page, but you might ask the designers if it's okay to leave it here.

Reply:
Added some django/css logic to make it appear if part appears